From 0cb7502af8d59ebd3fc11e24414f1cd3300e3c0b Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Thu, 1 Mar 2012 19:16:07 +0000
Subject: [PATCH 508/508] main: Enable quiet mode when bootspash is enabled.

Much like plymouth kicks in when 'splash' or 'splash=silent' is passed,
keep the systemd stuff quiet in this scenario too.

This prevents the bootsplash from kicking out and in at boot.
---
 src/main.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 60b41f1..65ff16d 100644
--- a/src/main.c
+++ b/src/main.c
@@ -378,7 +378,7 @@ static int parse_proc_cmdline_word(const char *word) {
                          "systemd.default_standard_error=null|tty|syslog|syslog+console|kmsg|kmsg+console|journal|journal+console\n"
                          "                                         Set default log error output for services\n");
 
-        } else if (streq(word, "quiet")) {
+        } else if (streq(word, "quiet") || streq(word, "splash") || streq(word, "splash=silent")) {
                 arg_show_status = false;
 #ifdef HAVE_SYSV_COMPAT
                 arg_sysv_console = false;
-- 
1.7.9.2

